src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <React.Fragment className="App">
+      <Display />
+      <ButtonPanel />
+    </React.Fragment>
```

Comment:
You have a warning in the console about the way you used the `React.Fragment`.  `React.Fragment` can only have `key` and `children` props.